src/aliceVision/sfmDataIO/viewIO.cpp

```diff
@@ -203,5 +201,32 @@ std::shared_ptr<camera::IntrinsicBase> getViewIntrinsic(const sfmData::View& vie
   return intrinsic;
 }
 
+const boost::filesystem::path viewPathFromFolders(const sfmData::View& view, const std::vector<std::string>& Folders)
```

Comment:
folders

Reply:
Fixed in https://github.com/alicevision/AliceVision/pull/806/commits/f7584167ca69f68bb430cb92284b5733df155fd7 .